index.html

```diff
@@ -1757,19 +1756,19 @@ <h4>Representing Time</h4>
 
         <p>
 This specification attempts to increase the number of universally recognized
-combinations of dates and times, and reduce the potentials for
-mis-interpretation of time values, by utilizing the
+combinations of dates and times, and reduce the potential for
+misinterpretation of time values, by utilizing the
 <code>dateTimeStamp</code> construction first established by the [<a
 data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification. In
-order to reduce mis-interpretations between different cultures, all times are
+order to reduce misinterpretations between different cultures, all times are
 specified as offsets against Universal Coordinated Time (UTC).
```

Comment:
A key problem when using offsets is that they do not represent the set of rules used to compute the offset. The rules, which are bound up in a "time zone", are subject to change and, as they are controlled by various national and sometimes sub-national governments and subject to local conditions, they _do_ change on a not-infrequent basis. Generally the change is small (by an hour here or there), so the impact is not perhaps critical to most applications. However, it does mean that a `validFrom` or `validTo` computed according to a local wall time under one set of time zone rules might start/expire earlier/later than expected due to a subsequent change to time zone rules. In particular, it could leave a gap of an hour or two between one credential's expiration and a new credential's validity if the new credential was computed with new time zone rules.

Standards for encoding the time zone into a time stamp are not yet "de jurre" and it seems unlikely that XML Schema will be revised to accommodate newer serializations incorporating zone IDs. For a peek into serialization types, see [this work in progress](https://w3c.github.io/timezone/#serializations) by I18N.

---

I don't see guidance to normalize dates to use UTC. That's a best practice which the specification can call out to users to avoid problems with time zone and it allows easier 1:1 comparison between separately generated values (even if it obscures what the local wall time was/will be)

---

> In order to reduce misinterpretations between different cultures, all times are
> specified as offsets against Universal Coordinated Time (UTC).

I think "cultures" is inappropriate here. Locale and culture are really orthogonal to the representation of time. The use of ISO8601-based serialization (and the preleptic Gregorian calendar inherent in that serialization) is what deals with cultural variation regarding the representation of time. The use of UTC and UTC offsets reduces the misinterpretation between _regions_.

Reply:
Note also that we're doing a review of this spec and there is a comment related to this PR [here](https://github.com/w3c/i18n-activity/issues/1729). This comment has not yet been reviewed by the I18N WG.

Reply:
Ok, focused on "regions" vs. "cultures" wrt. language around why this stuff is more complicated than it seems at first. Fixed in 01368c36ed23ef9e5d8421f8b350b728e6198d0d.

I also added a normative requirement to encode against Zulu time, but allowed for offsets. That seems to go against the guidance you gave above, primarily because I don't understand why encoding to an offset from Zulu is any different than encoding to Zulu. That is, whatever `dateTimeStamp` you create is going to be based off of your local time, isn't it? That is, you always lose the set of rules used to construct the time?

I'm happy to change it to something else, but I'm struggling w/ what the XML Schema 1.1: Part 2 spec says and what you're conveying (and the normative regex). I'm trying very hard to not modify the regex from what's used in  XML Schema 1.1: Part 2.